monkey/monkey_island/cc/services/zero_trust/scoutsuite/data_parsing/rule_path_building/ec2_rule_path_creator.py

```diff
@@ -0,0 +1,11 @@
+from monkey_island.cc.services.zero_trust.scoutsuite.consts.ec2_rules import EC2Rules
+from monkey_island.cc.services.zero_trust.scoutsuite.consts.service_consts import \
+    SERVICE_TYPES
+from monkey_island.cc.services.zero_trust.scoutsuite.data_parsing.rule_path_building.abstract_rule_path_creator import \
+    AbstractRulePathCreator
+
+
+class EC2RulePathCreator(AbstractRulePathCreator):
+
+    service_type = SERVICE_TYPES.EC2
+    supported_rules = EC2Rules
```

Comment:
This is not an instance of `List[Union[EC2Rules]]`

Reply:
not fixed

Reply:
Fixed it, if I understood the comment correctly.

Reply:
What I mean is, `EC2Rules` is of type `Type[EC2Rules]`
`AbstractRulePathCreator`'s `supported_rules`'s return type hint is `List`
These are not compatible.
What we should do as suggested in another comment, is to have the rule enums inherit from the same `Rule` enum class. Then, the correct return type hint would be `Type[Rule]` instead of `List`

Reply:
Sorry, don't know how I missed this :/ I'm not stoked with adding code for type hints (hints should compliment code, not the other way around), but makes sense in this case